src/ConnectedMode.UnitTests/Binding/CSharpVbBindingConfigTests.cs

```diff
@@ -47,42 +45,38 @@ public void Ctor_InvalidArgs()
         }
 
         [TestMethod]
-        public void GetSolutionLevelFilePaths_ReturnFilePaths()
```

Comment:
The property not longer exists on the interface -> this test is no longer required.